shared-module/sharpdisplay/SharpMemoryFramebuffer.c

```diff
@@ -271,7 +244,5 @@ const framebuffer_p_t sharpdisplay_framebuffer_proto = {
 };
 
 void common_hal_sharpdisplay_framebuffer_collect_ptrs(sharpdisplay_framebuffer_obj_t *self) {
-    gc_collect_ptr(self->framebuffer);
     gc_collect_ptr(self->bus);
-    gc_collect_ptr(self->bufinfo.buf);
```

Comment:
I'm not positive this is correct. Will moved memory still be as root pointers when off the heap?

Reply:
No, but does it need to? `gc_collect_ptr` will do nothing with a pointer that does not point into the heap. And as long as it does point into the heap, it’s not needed because it’s covered by the linked list. (In fact, even then `gc_collect_ptr` would do nothing with it, because it points into the middle of a GC block, after the `supervisor_allocation_node` header.)

But thanks for reminding me to take a second look at these, I may have made mistakes in similar cases.

Reply:
Ok! Sounds reasonable.